[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: linux
[INFO] os.detected.arch: x86_64
[INFO] os.detected.version: 5.15
[INFO] os.detected.version.major: 5
[INFO] os.detected.version.minor: 15
[INFO] os.detected.release: ubuntu
[INFO] os.detected.release.version: 20.04
[INFO] os.detected.release.like.ubuntu: true
[INFO] os.detected.release.like.debian: true
[INFO] os.detected.classifier: linux-x86_64
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Apache Ozone Integration Tests                                     [jar]
[INFO] Apache Ozone Mini Ozone Chaos Tests                                [jar]
[INFO] 
[INFO] --------------< org.apache.ozone:ozone-integration-test >---------------
[INFO] Building Apache Ozone Integration Tests 1.3.0-SNAPSHOT             [1/2]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-antrun-plugin:1.3:run (create-testdirs) @ ozone-integration-test ---
[INFO] Executing tasks
[INFO] Executed tasks
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (default-prepare-agent) @ ozone-integration-test ---
[INFO] argLine set to -javaagent:/home/runner/.m2/repository/org/jacoco/org.jacoco.agent/0.8.5/org.jacoco.agent-0.8.5-runtime.jar=destfile=/home/runner/work/ozone/ozone/hadoop-ozone/integration-test/target/jacoco.exec,includes=org.apache.hadoop.hdds.*:org.apache.hadoop.ozone.*:org.apache.hadoop.fs.ozone.*
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (banned-rocksdb-imports) @ ozone-integration-test ---
[INFO] 
[INFO] --- maven-remote-resources-plugin:1.5:process (default) @ ozone-integration-test ---
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ ozone-integration-test ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/runner/work/ozone/ozone/hadoop-ozone/integration-test/src/main/resources
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ ozone-integration-test ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ ozone-integration-test ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 21 resources
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ ozone-integration-test ---
[INFO] Compiling 12 source files to /home/runner/work/ozone/ozone/hadoop-ozone/integration-test/target/test-classes
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ ozone-integration-test ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.apache.hadoop.ozone.container.common.transport.server.ratis.TestCSMMetrics
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.394 s - in org.apache.hadoop.ozone.container.common.transport.server.ratis.TestCSMMetrics
[INFO] Running org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestBlockDeletion
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 305.159 s - in org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestBlockDeletion
[INFO] Running org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestCloseContainerHandler
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 37.491 s - in org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestCloseContainerHandler
[INFO] Running org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestRefreshVolumeUsageHandler
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 37.99 s - in org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestRefreshVolumeUsageHandler
[INFO] Running org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestDeleteContainerHandler
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 50.796 s - in org.apache.hadoop.ozone.container.common.statemachine.commandhandler.TestDeleteContainerHandler
[INFO] Running org.apache.hadoop.ozone.container.metrics.TestContainerMetrics
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 9.301 s - in org.apache.hadoop.ozone.container.metrics.TestContainerMetrics
[INFO] Running org.apache.hadoop.ozone.container.ozoneimpl.TestSecureOzoneContainer
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 17.686 s - in org.apache.hadoop.ozone.container.ozoneimpl.TestSecureOzoneContainer
[INFO] Running org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainer
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 123.596 s - in org.apache.hadoop.ozone.container.ozoneimpl.TestOzoneContainer
[INFO] Running org.apache.hadoop.ozone.container.TestECContainerRecovery
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 115.052 s - in org.apache.hadoop.ozone.container.TestECContainerRecovery
[INFO] Running org.apache.hadoop.ozone.container.TestContainerReplication
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 163.969 s - in org.apache.hadoop.ozone.container.TestContainerReplication
[INFO] Running org.apache.hadoop.ozone.container.server.TestSecureContainerServer
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 26.259 s - in org.apache.hadoop.ozone.container.server.TestSecureContainerServer
[INFO] Running org.apache.hadoop.ozone.parser.TestOzoneHARatisLogParser
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 59.401 s - in org.apache.hadoop.ozone.parser.TestOzoneHARatisLogParser
[INFO] Running org.apache.hadoop.ozone.dn.scrubber.TestDataScrubber
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 50.171 s - in org.apache.hadoop.ozone.dn.scrubber.TestDataScrubber
[INFO] Running org.apache.hadoop.ozone.dn.ratis.TestDnRatisLogParser
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 25.051 s - in org.apache.hadoop.ozone.dn.ratis.TestDnRatisLogParser
[INFO] Running org.apache.hadoop.ozone.dn.volume.TestDatanodeHddsVolumeFailureDetection
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 88.854 s - in org.apache.hadoop.ozone.dn.volume.TestDatanodeHddsVolumeFailureDetection
[INFO] Running org.apache.hadoop.ozone.dn.volume.TestDatanodeHddsVolumeFailureToleration
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 43.039 s - in org.apache.hadoop.ozone.dn.volume.TestDatanodeHddsVolumeFailureToleration
[INFO] Running org.apache.hadoop.ozone.TestStorageContainerManager
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.apache.hadoop.ozone.TestStorageContainerManager (file:/home/runner/work/ozone/ozone/hadoop-ozone/integration-test/target/test-classes/) to field java.lang.reflect.Field.modifiers
[ERROR] WARNING: Please consider reporting this to the maintainers of org.apache.hadoop.ozone.TestStorageContainerManager
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 399.912 s - in org.apache.hadoop.ozone.TestStorageContainerManager
[INFO] Running org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] Tests run: 11, Failures: 0, Errors: 7, Skipped: 0, Time elapsed: 50.441 s <<< FAILURE! - in org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOmReInit  Time elapsed: 8.9 s  <<< ERROR!
java.net.BindException: Problem binding to [localhost:1202] java.net.BindException: Address already in use; For more details see:  http://wiki.apache.org/hadoop/BindException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:809)
	at org.apache.hadoop.ipc.Server.bind(Server.java:640)
	at org.apache.hadoop.ipc.Server$Listener.<init>(Server.java:1225)
	at org.apache.hadoop.ipc.Server.<init>(Server.java:3117)
	at org.apache.hadoop.ipc.RPC$Server.<init>(RPC.java:1062)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server.<init>(ProtobufRpcEngine2.java:464)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.<init>(ProtobufRpcEngine.java:434)
	at org.apache.hadoop.ipc.ProtobufRpcEngine.getServer(ProtobufRpcEngine.java:361)
	at org.apache.hadoop.ipc.RPC$Builder.build(RPC.java:853)
	at org.apache.hadoop.ozone.om.OzoneManager.startRpcServer(OzoneManager.java:1150)
	at org.apache.hadoop.ozone.om.OzoneManager.getRpcServer(OzoneManager.java:1120)
	at org.apache.hadoop.ozone.om.OzoneManager.<init>(OzoneManager.java:598)
	at org.apache.hadoop.ozone.om.OzoneManager.createOm(OzoneManager.java:662)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOmReInit(TestSecureOzoneCluster.java:761)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.apache.hadoop.ipc.Server.bind(Server.java:623)
	... 27 more

[ERROR] org.apache.hadoop.ozone.TestSecureOzoneCluster.testGetSetRevokeS3Secret  Time elapsed: 2.326 s  <<< ERROR!
java.net.BindException: Problem binding to [localhost:1202] java.net.BindException: Address already in use; For more details see:  http://wiki.apache.org/hadoop/BindException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:809)
	at org.apache.hadoop.ipc.Server.bind(Server.java:640)
	at org.apache.hadoop.ipc.Server$Listener.<init>(Server.java:1225)
	at org.apache.hadoop.ipc.Server.<init>(Server.java:3117)
	at org.apache.hadoop.ipc.RPC$Server.<init>(RPC.java:1062)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server.<init>(ProtobufRpcEngine2.java:464)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.<init>(ProtobufRpcEngine.java:434)
	at org.apache.hadoop.ipc.ProtobufRpcEngine.getServer(ProtobufRpcEngine.java:361)
	at org.apache.hadoop.ipc.RPC$Builder.build(RPC.java:853)
	at org.apache.hadoop.ozone.om.OzoneManager.startRpcServer(OzoneManager.java:1150)
	at org.apache.hadoop.ozone.om.OzoneManager.getRpcServer(OzoneManager.java:1120)
	at org.apache.hadoop.ozone.om.OzoneManager.<init>(OzoneManager.java:598)
	at org.apache.hadoop.ozone.om.OzoneManager.createOm(OzoneManager.java:662)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.setupOm(TestSecureOzoneCluster.java:621)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.testGetSetRevokeS3Secret(TestSecureOzoneCluster.java:628)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.apache.hadoop.ipc.Server.bind(Server.java:623)
	... 28 more

[ERROR] org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOmInitSuccess  Time elapsed: 5.297 s  <<< ERROR!
java.net.BindException: Problem binding to [localhost:1202] java.net.BindException: Address already in use; For more details see:  http://wiki.apache.org/hadoop/BindException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:809)
	at org.apache.hadoop.ipc.Server.bind(Server.java:640)
	at org.apache.hadoop.ipc.Server$Listener.<init>(Server.java:1225)
	at org.apache.hadoop.ipc.Server.<init>(Server.java:3117)
	at org.apache.hadoop.ipc.RPC$Server.<init>(RPC.java:1062)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server.<init>(ProtobufRpcEngine2.java:464)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.<init>(ProtobufRpcEngine.java:434)
	at org.apache.hadoop.ipc.ProtobufRpcEngine.getServer(ProtobufRpcEngine.java:361)
	at org.apache.hadoop.ipc.RPC$Builder.build(RPC.java:853)
	at org.apache.hadoop.ozone.om.OzoneManager.startRpcServer(OzoneManager.java:1150)
	at org.apache.hadoop.ozone.om.OzoneManager.getRpcServer(OzoneManager.java:1120)
	at org.apache.hadoop.ozone.om.OzoneManager.<init>(OzoneManager.java:598)
	at org.apache.hadoop.ozone.om.OzoneManager.createOm(OzoneManager.java:662)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOmInitSuccess(TestSecureOzoneCluster.java:797)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.apache.hadoop.ipc.Server.bind(Server.java:623)
	... 27 more

[ERROR] org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOmInitializationSuccess  Time elapsed: 4.698 s  <<< ERROR!
java.net.BindException: Problem binding to [localhost:1202] java.net.BindException: Address already in use; For more details see:  http://wiki.apache.org/hadoop/BindException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:809)
	at org.apache.hadoop.ipc.Server.bind(Server.java:640)
	at org.apache.hadoop.ipc.Server$Listener.<init>(Server.java:1225)
	at org.apache.hadoop.ipc.Server.<init>(Server.java:3117)
	at org.apache.hadoop.ipc.RPC$Server.<init>(RPC.java:1062)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server.<init>(ProtobufRpcEngine2.java:464)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.<init>(ProtobufRpcEngine.java:434)
	at org.apache.hadoop.ipc.ProtobufRpcEngine.getServer(ProtobufRpcEngine.java:361)
	at org.apache.hadoop.ipc.RPC$Builder.build(RPC.java:853)
	at org.apache.hadoop.ozone.om.OzoneManager.startRpcServer(OzoneManager.java:1150)
	at org.apache.hadoop.ozone.om.OzoneManager.getRpcServer(OzoneManager.java:1120)
	at org.apache.hadoop.ozone.om.OzoneManager.<init>(OzoneManager.java:598)
	at org.apache.hadoop.ozone.om.OzoneManager.createOm(OzoneManager.java:662)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.setupOm(TestSecureOzoneCluster.java:621)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOmInitializationSuccess(TestSecureOzoneCluster.java:462)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.apache.hadoop.ipc.Server.bind(Server.java:623)
	... 28 more

[ERROR] org.apache.hadoop.ozone.TestSecureOzoneCluster.testAccessControlExceptionOnClient  Time elapsed: 3.798 s  <<< ERROR!
java.net.BindException: Problem binding to [localhost:1202] java.net.BindException: Address already in use; For more details see:  http://wiki.apache.org/hadoop/BindException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:809)
	at org.apache.hadoop.ipc.Server.bind(Server.java:640)
	at org.apache.hadoop.ipc.Server$Listener.<init>(Server.java:1225)
	at org.apache.hadoop.ipc.Server.<init>(Server.java:3117)
	at org.apache.hadoop.ipc.RPC$Server.<init>(RPC.java:1062)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server.<init>(ProtobufRpcEngine2.java:464)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.<init>(ProtobufRpcEngine.java:434)
	at org.apache.hadoop.ipc.ProtobufRpcEngine.getServer(ProtobufRpcEngine.java:361)
	at org.apache.hadoop.ipc.RPC$Builder.build(RPC.java:853)
	at org.apache.hadoop.ozone.om.OzoneManager.startRpcServer(OzoneManager.java:1150)
	at org.apache.hadoop.ozone.om.OzoneManager.getRpcServer(OzoneManager.java:1120)
	at org.apache.hadoop.ozone.om.OzoneManager.<init>(OzoneManager.java:598)
	at org.apache.hadoop.ozone.om.OzoneManager.createOm(OzoneManager.java:662)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.setupOm(TestSecureOzoneCluster.java:621)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.testAccessControlExceptionOnClient(TestSecureOzoneCluster.java:479)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.apache.hadoop.ipc.Server.bind(Server.java:623)
	... 28 more

[ERROR] org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOMInitializationFailure  Time elapsed: 4.411 s  <<< ERROR!
java.net.BindException: Problem binding to [localhost:1202] java.net.BindException: Address already in use; For more details see:  http://wiki.apache.org/hadoop/BindException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:809)
	at org.apache.hadoop.ipc.Server.bind(Server.java:640)
	at org.apache.hadoop.ipc.Server$Listener.<init>(Server.java:1225)
	at org.apache.hadoop.ipc.Server.<init>(Server.java:3117)
	at org.apache.hadoop.ipc.RPC$Server.<init>(RPC.java:1062)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server.<init>(ProtobufRpcEngine2.java:464)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.<init>(ProtobufRpcEngine.java:434)
	at org.apache.hadoop.ipc.ProtobufRpcEngine.getServer(ProtobufRpcEngine.java:361)
	at org.apache.hadoop.ipc.RPC$Builder.build(RPC.java:853)
	at org.apache.hadoop.ozone.om.OzoneManager.startRpcServer(OzoneManager.java:1150)
	at org.apache.hadoop.ozone.om.OzoneManager.getRpcServer(OzoneManager.java:1120)
	at org.apache.hadoop.ozone.om.OzoneManager.<init>(OzoneManager.java:598)
	at org.apache.hadoop.ozone.om.OzoneManager.createOm(OzoneManager.java:662)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.setupOm(TestSecureOzoneCluster.java:621)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.testSecureOMInitializationFailure(TestSecureOzoneCluster.java:445)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.apache.hadoop.ipc.Server.bind(Server.java:623)
	... 28 more

[ERROR] org.apache.hadoop.ozone.TestSecureOzoneCluster.testDelegationTokenRenewal  Time elapsed: 1.916 s  <<< ERROR!
java.net.BindException: Problem binding to [localhost:1202] java.net.BindException: Address already in use; For more details see:  http://wiki.apache.org/hadoop/BindException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:809)
	at org.apache.hadoop.ipc.Server.bind(Server.java:640)
	at org.apache.hadoop.ipc.Server$Listener.<init>(Server.java:1225)
	at org.apache.hadoop.ipc.Server.<init>(Server.java:3117)
	at org.apache.hadoop.ipc.RPC$Server.<init>(RPC.java:1062)
	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Server.<init>(ProtobufRpcEngine2.java:464)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server.<init>(ProtobufRpcEngine.java:434)
	at org.apache.hadoop.ipc.ProtobufRpcEngine.getServer(ProtobufRpcEngine.java:361)
	at org.apache.hadoop.ipc.RPC$Builder.build(RPC.java:853)
	at org.apache.hadoop.ozone.om.OzoneManager.startRpcServer(OzoneManager.java:1150)
	at org.apache.hadoop.ozone.om.OzoneManager.getRpcServer(OzoneManager.java:1120)
	at org.apache.hadoop.ozone.om.OzoneManager.<init>(OzoneManager.java:598)
	at org.apache.hadoop.ozone.om.OzoneManager.createOm(OzoneManager.java:662)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.setupOm(TestSecureOzoneCluster.java:621)
	at org.apache.hadoop.ozone.TestSecureOzoneCluster.testDelegationTokenRenewal(TestSecureOzoneCluster.java:542)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.apache.hadoop.ipc.Server.bind(Server.java:623)
	... 28 more

[INFO] Running org.apache.hadoop.ozone.TestMiniOzoneCluster
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 90.303 s - in org.apache.hadoop.ozone.TestMiniOzoneCluster
[INFO] Running org.apache.hadoop.ozone.TestMiniOzoneOMHACluster
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 43.942 s - in org.apache.hadoop.ozone.TestMiniOzoneOMHACluster
[INFO] Running org.apache.hadoop.ozone.fsck.TestContainerMapper
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 45.479 s - in org.apache.hadoop.ozone.fsck.TestContainerMapper
[INFO] Running org.apache.hadoop.ozone.TestContainerOperations
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 46.54 s - in org.apache.hadoop.ozone.TestContainerOperations
[INFO] Running org.apache.hadoop.ozone.freon.TestRandomKeyGenerator
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 197.994 s - in org.apache.hadoop.ozone.freon.TestRandomKeyGenerator
[INFO] Running org.apache.hadoop.ozone.freon.TestFreonWithDatanodeFastRestart
[WARNING] Tests run: 1, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.016 s - in org.apache.hadoop.ozone.freon.TestFreonWithDatanodeFastRestart
[INFO] Running org.apache.hadoop.ozone.freon.TestDataValidateWithDummyContainers
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 58.251 s - in org.apache.hadoop.ozone.freon.TestDataValidateWithDummyContainers
[INFO] Running org.apache.hadoop.ozone.freon.TestOmBucketReadWriteFileOps
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 67.559 s - in org.apache.hadoop.ozone.freon.TestOmBucketReadWriteFileOps
[INFO] Running org.apache.hadoop.ozone.freon.TestFreonWithPipelineDestroy
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 59.239 s - in org.apache.hadoop.ozone.freon.TestFreonWithPipelineDestroy
[INFO] Running org.apache.hadoop.ozone.freon.TestHadoopDirTreeGeneratorWithFSO
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 78.617 s - in org.apache.hadoop.ozone.freon.TestHadoopDirTreeGeneratorWithFSO
[INFO] Running org.apache.hadoop.ozone.freon.TestDataValidateWithSafeByteOperations
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 67.469 s - in org.apache.hadoop.ozone.freon.TestDataValidateWithSafeByteOperations
[INFO] Running org.apache.hadoop.ozone.freon.TestOmBucketReadWriteKeyOps
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 73.252 s - in org.apache.hadoop.ozone.freon.TestOmBucketReadWriteKeyOps
[INFO] Running org.apache.hadoop.ozone.freon.TestDataValidateWithUnsafeByteOperations
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 68.427 s - in org.apache.hadoop.ozone.freon.TestDataValidateWithUnsafeByteOperations
[INFO] Running org.apache.hadoop.ozone.freon.TestHadoopDirTreeGenerator
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 74.437 s - in org.apache.hadoop.ozone.freon.TestHadoopDirTreeGenerator
[INFO] Running org.apache.hadoop.ozone.freon.TestHadoopNestedDirGenerator
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 61.856 s - in org.apache.hadoop.ozone.freon.TestHadoopNestedDirGenerator
[INFO] Running org.apache.hadoop.ozone.TestContainerStateMachineIdempotency
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 52.317 s - in org.apache.hadoop.ozone.TestContainerStateMachineIdempotency
[INFO] Running org.apache.hadoop.ozone.TestOzoneConfigurationFields
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.421 s - in org.apache.hadoop.ozone.TestOzoneConfigurationFields
[INFO] Running org.apache.hadoop.ozone.recon.TestReconWithOzoneManagerHA
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 43.721 s - in org.apache.hadoop.ozone.recon.TestReconWithOzoneManagerHA
[INFO] Running org.apache.hadoop.ozone.recon.TestReconTasks
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 68.55 s - in org.apache.hadoop.ozone.recon.TestReconTasks
[INFO] Running org.apache.hadoop.ozone.recon.TestReconScmHASnapshot
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 55.858 s - in org.apache.hadoop.ozone.recon.TestReconScmHASnapshot
[INFO] Running org.apache.hadoop.ozone.recon.TestReconAsPassiveScm
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 95.884 s - in org.apache.hadoop.ozone.recon.TestReconAsPassiveScm
[INFO] Running org.apache.hadoop.ozone.recon.TestReconScmSnapshot
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 46.484 s - in org.apache.hadoop.ozone.recon.TestReconScmSnapshot
[INFO] Running org.apache.hadoop.ozone.recon.TestReconWithOzoneManagerFSO
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 41.116 s - in org.apache.hadoop.ozone.recon.TestReconWithOzoneManagerFSO
[INFO] Running org.apache.hadoop.ozone.recon.TestReconWithOzoneManager
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 31.315 s - in org.apache.hadoop.ozone.recon.TestReconWithOzoneManager
[INFO] Running org.apache.hadoop.ozone.TestContainerBalancerOperations
[WARNING] Tests run: 1, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.011 s - in org.apache.hadoop.ozone.TestContainerBalancerOperations
[INFO] Running org.apache.hadoop.ozone.shell.TestOzoneDatanodeShell
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.734 s - in org.apache.hadoop.ozone.shell.TestOzoneDatanodeShell
[INFO] Running org.apache.hadoop.ozone.shell.TestScmAdminHA
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 48.471 s - in org.apache.hadoop.ozone.shell.TestScmAdminHA
[INFO] Running org.apache.hadoop.ozone.shell.TestOzoneTenantShell
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 14.286 s - in org.apache.hadoop.ozone.shell.TestOzoneTenantShell
[INFO] Running org.apache.hadoop.ozone.shell.TestOzoneShellHAWithFSO
[INFO] Tests run: 13, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 85.432 s - in org.apache.hadoop.ozone.shell.TestOzoneShellHAWithFSO
[INFO] Running org.apache.hadoop.ozone.shell.TestOzoneShellHA
[INFO] Tests run: 13, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 80.93 s - in org.apache.hadoop.ozone.shell.TestOzoneShellHA
[INFO] Running org.apache.hadoop.ozone.shell.TestResetDeletedBlockRetryCountShell
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 74.236 s - in org.apache.hadoop.ozone.shell.TestResetDeletedBlockRetryCountShell
[INFO] Running org.apache.hadoop.ozone.shell.TestNSSummaryAdmin
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.jooq.tools.reflect.Reflect (file:/home/runner/.m2/repository/org/jooq/jooq/3.11.10/jooq-3.11.10.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.jooq.tools.reflect.Reflect
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.754 s - in org.apache.hadoop.ozone.shell.TestNSSummaryAdmin
[INFO] Running org.apache.hadoop.ozone.TestDelegationToken
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.917 s - in org.apache.hadoop.ozone.TestDelegationToken
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   TestSecureOzoneCluster.testAccessControlExceptionOnClient:479->setupOm:621 » Bind
[ERROR]   TestSecureOzoneCluster.testDelegationTokenRenewal:542->setupOm:621 » Bind Prob...
[ERROR]   TestSecureOzoneCluster.testGetSetRevokeS3Secret:628->setupOm:621 » Bind Proble...
[ERROR]   TestSecureOzoneCluster.testSecureOMInitializationFailure:445->setupOm:621 » Bind
[ERROR]   TestSecureOzoneCluster.testSecureOmInitSuccess:797 » Bind Problem binding to [...
[ERROR]   TestSecureOzoneCluster.testSecureOmInitializationSuccess:462->setupOm:621 » Bind
[ERROR]   TestSecureOzoneCluster.testSecureOmReInit:761 » Bind Problem binding to [local...
[INFO] 
[ERROR] Tests run: 139, Failures: 0, Errors: 7, Skipped: 2
[INFO] 
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Skipping Apache Ozone Mini Ozone Chaos Tests
[INFO] This project has been banned from the build due to previous failures.
[INFO] ------------------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache Ozone Integration Tests 1.3.0-SNAPSHOT:
[INFO] 
[INFO] Apache Ozone Integration Tests ..................... FAILURE [58:26 min]
[INFO] Apache Ozone Mini Ozone Chaos Tests ................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  58:27 min
[INFO] Finished at: 2022-08-23T01:56:34Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project ozone-integration-test: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/runner/work/ozone/ozone/hadoop-ozone/integration-test/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
